products/views.py

```diff
@@ -1,3 +1,28 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http            import JsonResponse
+from django.views           import View
+
+from products.models        import Product, Image
+
+class ProductDetailView(View):
+    def get(self, request, product_id):
+        try: 
+            product = Product.objects.get(id=product_id)
+            image   = Image.objects.filter(id=product_id)
+  
+            result = [
+                        {
+                            'name'                : product.name,
+                            'price'               : product.price,
+                            'brand'               : product.brand,
+                            'description'         : product.description,
+                            'thumbnail_image_url' : product.thumbnail_image_url,
+                            'image_url'           : image[0].url
```

Comment:
위의 product 객체만 사용해서 image_url을 가져올 수 있도록 코드를 수정해주세요~
```suggestion
                            'image_url' : product....
```

Reply:
적용하였습니다!